LibsNoSamples.sln

```diff
@@ -1,136 +1,173 @@
+		src\MSAL.Frameworks.props = src\MSAL.Frameworks.props
+		build\SolutionWideAnalyzerConfig.ruleset = build\SolutionWideAnalyzerConfig.ruleset
+	EndProjectSection
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Identity.Test.Unit.netcore", "tests\Microsoft.Identity.Test.Unit.netcore\Microsoft.Identity.Test.Unit.netcore.csproj", "{AF0E7BEC-90A3-46B4-AEDC-4162D7873AE8}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Identity.Test.Unit.net45", "tests\Microsoft.Identity.Test.Unit.net45\Microsoft.Identity.Test.Unit.net45.csproj", "{DA8E7933-0A56-4668-9786-ECECA3D16944}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Identity.Test.Common", "tests\Microsoft.Identity.Test.Common\Microsoft.Identity.Test.Common.csproj", "{30287EC8-BD85-41CC-BDCB-9DBF9EFBCC98}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Identity.Test.LabInfrastructure", "tests\Microsoft.Identity.Test.LabInfrastructure\Microsoft.Identity.Test.LabInfrastructure.csproj", "{E22918A8-8FE6-47D2-B65E-34436918C5D7}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Identity.Test.SideBySide", "tests\Microsoft.Identity.Test.SideBySide\Microsoft.Identity.Test.SideBySide.csproj", "{D3033A40-F2C2-4495-BC9F-C3C6D497D1CC}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Identity.Client.Ref", "src\Microsoft.Identity.Client.Ref\Microsoft.Identity.Client.Ref.csproj", "{528C5488-C0D8-45C3-BEAC-3E52C22631C9}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ref", "ref", "{528F34D9-FF6B-4AAE-8367-182EC19DAB5F}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CommonCache.Test.AdalV3", "tests\CacheCompat\CommonCache.Test.AdalV3\CommonCache.Test.AdalV3.csproj", "{A8C2D397-BA21-4BD2-93AA-801B0E0804BC}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "CacheCompat", "CacheCompat", "{7930A443-1981-49CD-8AF5-02E25F3A7FF3}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CommonCache.Test.AdalV4", "tests\CacheCompat\CommonCache.Test.AdalV4\CommonCache.Test.AdalV4.csproj", "{70DFE67C-D2D1-4BE4-B785-F4445C6F652D}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CommonCache.Test.Common", "tests\CacheCompat\CommonCache.Test.Common\CommonCache.Test.Common.csproj", "{21B0D02E-8E52-4ECD-BFD3-5F14FCB73703}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CommonCache.Test.MsalV2", "tests\CacheCompat\CommonCache.Test.MsalV2\CommonCache.Test.MsalV2.csproj", "{8672FC71-00DE-491D-BCEB-101A2A7887E6}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CommonCache.Test.Unit", "tests\CacheCompat\CommonCache.Test.Unit\CommonCache.Test.Unit.csproj", "{CFC2232F-3276-42A4-B8E3-F03B6F8BAA82}"
+	ProjectSection(ProjectDependencies) = postProject
+		{CA3F8F2E-7DF2-45A6-BC81-5DA5692FCC11} = {CA3F8F2E-7DF2-45A6-BC81-5DA5692FCC11}

```

Comment:
seems like these proj updates should also happen in other solution files?

Reply:
done.